lib/help_scout.rb

```diff
@@ -1,6 +1,8 @@
 require "help_scout/version"
 require "httparty"
 
+class HelpScout::ValidationError < StandardError; end
```

Comment:
You can also define this inside the `HelpScout` class, I think I've seen it be done like that more often.